### PR TITLE
Added E.ON HUNGARY specification

### PR DESCRIPTION
**Pull request by balazs92117**
I have added new specification for E.ON service provider of Hungary.

Unfortunately the #80 issue is still exists, so I modified the clients/socket_.py
Until the service provider does not send a "normal" message instead of `?` signs, we can't further test the issue.

### CODE REVIEW

**Comment by balazs92117** [comment]
@ndokter Can you take a look at this pls?

File: dsmr_parser/clients/socket_.py [hunk 1]
```diff
@@ -43,7 +43,12 @@ def read(self):
                     continue
 
                 for data in lines:
-                    self.telegram_buffer.append(data.decode('ascii'))
+                    try:
+                        self.telegram_buffer.append(data.decode('ascii'))
+                    except UnicodeDecodeError:
+                        # Some garbage came through the channel
+                        # E.g.: Happens at EON_HUNGARY, but only once at the start of the socket.
+                        pass
```
Comment (ndokter) [inline dsmr_parser/clients/socket_.py:51]: Maybe add a error log here as well:
```
logger.error('Failed to parse telegram due to unicode decode error')
```

**Comment by ndokter** [comment]
I would add a test like https://github.com/balazs92117/dsmr_parser/blob/eff0a5e1fc7065af902084e356fe6208fb563482/test/test_parse_v5.py maybe called test_parse_v5_hungary.py

Where we can validate that a complete eon/hungary telegram parsees into all expected values

File: dsmr_parser/clients/socket_.py (hunk 1 above)
Comment (balazs92117) [reply dsmr_parser/clients/socket_.py]: Added the error logging.

**Comment by balazs92117** [comment]
@ndokter
I added tests for the new parser. I called it test_parse_v5_eon_hungary.py as we have multiple service providers, not just "E.ON"

**Comment by ndokter** [comment]
Thanks for your work, i will make a new build soon

**Comment by balazs92117** [comment]
Thank you! :)

**Comment by ndokter** [comment]
Hi im sorry i noticed after merging that tests are breaking due to the belgium keys conflicting. Maybe these need to be renamed to a more general key or something else needs to be thought of for `dsmr_parser.obis_name_mapping.EN`. Something like a translation per dsmr specification in here `dsmr_parser.telegram_specifications`

**Pull request merged**